bootstrap/kubeadm/main.go

```diff
@@ -136,8 +135,11 @@ func main() {
 		RetryPeriod:        &leaderElectionRetryPeriod,
 		Namespace:          watchNamespace,
 		SyncPeriod:         &syncPeriod,
-		NewClient:          newClientFunc,
-		Port:               webhookPort,
+		NewClient: util.DelegatingClientFuncWithUncached(
+			&corev1.ConfigMap{},
```

Comment:
Do we need to add ConfigMapList and SecretList? (here and everywhere)

Reply:
Or make the new client code smarter?

Reply:
We should probably be explicit, unless there is a safe way to get a list-type GVK from a scheme?